### PR TITLE
Tweaks to Accessibility page in French

### PR DESCRIPTION
**Pull request by anikbrazeau**
Got review from translation to improve the Accessibility page content

### CODE REVIEW

File: app/translations/csv/fr.csv [hunk 1]
```diff
@@ -1002,28 +1002,28 @@
...
+"We are committed to meeting your needs no matter how you use the service. We understand that people may have different needs for accommodation. To make sure we can be inclusive of everyone, we want to learn from you.","Nous nous engageons à répondre à vos besoins, quelle que soit la façon dont vous utilisez le service. Nous comprenons que les gens peuvent avoir des besoins différents en matière d’adaptation. Pour nous assurer que toute personne est incluse, nous voulons apprendre de vos expériences."
 "Tell us how we’re doing","Donnez-nous votre avis"
-"To tell us about accessibility issues in GC Notify, you can:","Faites-nous part de problèmes d’accessibilité dans GC Notification&nbsp;:"
+"To tell us about accessibility issues in GC Notify, you can:","Pour nous faire part de problèmes d’accessibilité dans GC Notification&nbsp;:"
 "Reach out to us through our ","Utilisez notre "
-"contact form","formulaire de contact"
+"contact form","formulaire «&nbsp;Nous joindre&nbsp;»"
```
Comment (sastels) [inline app/translations/csv/fr.csv:1018]: nbsp ftw!

**Comment by anikbrazeau** [comment]
[Write A11y statement page](https://trello.com/c/v7ehHmoV/697-write-a11y-statement-page)

**Review by sastels — APPROVED** [review]
🚀

**Comment by anikbrazeau** [comment]
[Update current peripheral content for accuracy and scannability -- Terms, Privacy, Security](https://trello.com/c/h5iWftWx/414-update-current-peripheral-content-for-accuracy-and-scannability-terms-privacy-security)

**Pull request merged**